modules/loki-deployment-sizing.adoc

```diff
@@ -1,20 +1,15 @@
 // Module is included in the following assemblies:
 //cluster-logging-loki.adoc
-:_content-type: CONCEPT
-[id="about-logging-loki_{context}"]
-= About the LokiStack
-
-In {logging} documentation, *LokiStack* refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. *Loki* refers to the log store as either the individual component or an external store.
 
-Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}. Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion, and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly. As with Elasticsearch, you can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].
+:_content-type: CONCEPT
+[id="loki-deployment-sizing_{context}"]
+= Deployment Sizing
 
-[id="deployment-sizing_{context}"]
-== Deployment Sizing
 Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
```

Comment:
We only support 1x.extra-small, 1x.small and 1x.medium. 
Not sure why we have <N> is number of instances here since it will always be '1'. I think we should have it as 1x.<size> where <size> specifies performance capabilities. That's just my thinking :)

Reply:
@kabirbhartiRH since this content wasn't actually modified in this PR, would you mind please opening it as a bug in the docs Jira to fix this later? I want to try to keep the scope of the changes to the bug being fixed, especially since this PR has been open for a while now 🙂